core/src/main/java/org/apache/spark/shuffle/api/ShuffleExecutorComponents.java

```diff
@@ -39,17 +39,15 @@
   /**
    * Called once per map task to create a writer that will be responsible for persisting all the
    * partitioned bytes written by that map task.
-   *  @param shuffleId Unique identifier for the shuffle the map task is a part of
-   * @param mapId Within the shuffle, the identifier of the map task
+   * @param shuffleId Unique identifier for the shuffle the map task is a part of
    * @param mapTaskAttemptId Identifier of the task attempt. Multiple attempts of the same map task
- *                         with the same (shuffleId, mapId) pair can be distinguished by the
- *                         different values of mapTaskAttemptId.
+   *                         with the same (shuffleId, mapId) pair can be distinguished by the
+   *                         different values of mapTaskAttemptId.
    * @param numPartitions The number of partitions that will be written by the map task. Some of
-*                      these partitions may be empty.
+   *                      these partitions may be empty.
    */
   ShuffleMapOutputWriter createMapOutputWriter(
       int shuffleId,
-      int mapId,
       long mapTaskAttemptId,
```

Comment:
shall we name it `mapId`? To be consistent with the codebase.

Reply:
The original mapId still used in mapOutputTracker and scheduler, I doubt anybody will confused by these two ids use same name?

Reply:
I changed all `mapTaskAttemptId` stuff to `mapId` in 539d725.
So after the change, the mapId is the unique id for a map task. If we think it's confused to have a mapId represent the map index within a stage or a task set, `mapIndex` maybe a much better name.